src/lib/spsp.js

```diff
@@ -87,7 +94,8 @@ const quoteSource = (plugin, receiver, amount) => {
 const quoteDestination = (plugin, receiver, amount) => {
   return co(function * () {
     const spsp = yield _querySPSP(receiver)
-    const quote = yield _quote(plugin, spsp, amount)
+    // quote by the destination amount, leaving source amount unspecified
+    const quote = yield _quote(plugin, spsp, undefined, amount)
```

Comment:
I think it would probably be better to pass an object with named parameters to avoid this kind of bug